app.js

```diff
@@ -1,82 +1,84 @@
-const Store = {
-  getBooks() {
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  static displayBooks() {
+    const books = Book.getBooks();
+    books.forEach((book) => Book.addBookToList(book));
+  }
+
+  static addBookToList(book) {
+    const list = document.querySelector('#book-list');
+
+    const row = document.createElement('tr');
+
+    row.innerHTML = `
+    <td>"${book.title}" by ${book.author}</td>
+    <td class="d-none">${book.author}</td>
+    <td><a href="#" class="btn btn-sm btn-danger delete float-end">remove</a></td>
+    `;
+    list.appendChild(row);
+  }
+
+  static deleteBook(el) {
+    if (el.classList.contains('delete')) {
+      el.parentElement.parentElement.remove();
+    }
+  }
+
+  static clearField() {
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
+  }
+
+  static getBooks() {
     let books;
     if (localStorage.getItem('books') === null) {
       books = [];
     } else {
       books = JSON.parse(localStorage.getItem('books'));
     }
     return books;
-  },
+  }
 
-  addBook(book) {
-    const books = Store.getBooks();
+  static addBook(book) {
+    const books = Book.getBooks();
     books.push(book);
```

Comment:
# Optional:  Please use `concat()` method because it doesn't mutate the array
- The `push()` method is a generic method similar to call() or apply(). It will mutate your array (or object), pushing a new value into it.
- The `concat()` method returns a new array, with the values merged. This also avoids mutation side effects.
- [Read more here.](https://stackoverflow.com/questions/48428044/why-use-concat-instead-of-push-in-this-instance)